stable/mariadb/values.yaml

```diff
@@ -25,3 +25,10 @@ imageTag: 10.1.14-r3
 ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#creating-a-database-on-first-run
 ##
 # mariadbDatabase:
+
+## Persist data to a persitent volume
+persistence:
+    enabled: true
```

Comment:
other YAML files use 2 spaces for indentation, can we change this to 2 spaces too?